src/esp/scene/SceneNode.cpp

```diff
@@ -16,16 +18,61 @@ SceneNode::SceneNode(SceneNode& parent)
   setId(parent.getId());
   setCachedTransformations(Mn::SceneGraph::CachedTransformation::Absolute);
   absoluteTransformation_ = absoluteTransformation();
+  nodeSensorSuite_ = new esp::sensor::SensorSuite(*this);
+  subtreeSensorSuite_ = new esp::sensor::SensorSuite(*this);
 }
 
 SceneNode::SceneNode(MagnumScene& parentNode)
     : Mn::SceneGraph::AbstractFeature3D{*this} {
   setParent(&parentNode);
   setCachedTransformations(Mn::SceneGraph::CachedTransformation::Absolute);
   absoluteTransformation_ = absoluteTransformation();
+  nodeSensorSuite_ = new esp::sensor::SensorSuite(*this);
+  subtreeSensorSuite_ = new esp::sensor::SensorSuite(*this);
+}
+
+SceneNode::~SceneNode() {
```

Comment:
I just had a meeting with @mosra this morning.
He found the root cause causing `node().parent()` is nullptr.
After he push the fixes, he will bump the habitat to a newer version of corrade and magnum.
Then we will move the code back to the `~Sensor()`.
Yeah, it is a bit inconvenience, and annoying. But the reason is that: We inserted a lot of logic in the Sensor's ctor. Whatever constructed in the Sensor's ctor, should be deconstructed in the Sensor's dtor. In terms of design, it is MUCH LESS error-prone.